index.html

```diff
@@ -4714,14 +5028,13 @@ <h3 id="Warrior">Warrior</h3>
             <tr>
                <td>7
                </td>
-               <td>Sundering Strike
+               <td>...The Harder They Fall
                </td>
-               <td><strong>Utterly break an enemy's defences.</strong>
+               <td><strong>Use a creature's size against it.</strong>
                   <p>
-                     Make a melee weapon attack against a creature and on a hit the creature is
-                     <a class="condition" data-tooltip-container>marked</a> and its AC is set to 8. Deal additional damage equal to half the 
-                     AC reduction the creature suffers. For example, a reduction from 15 AC to 8 is a 
-                     reduction of 7 and therefore results in 3 additional damage.
+                     Make a melee weapon attack against a creature and on a hit the creature suffers
```

Comment:
Don't love this one - themeing implies it's a kill move whereas it gets much less effective throughout the fight.  Might lead to "no don't do that you'll reduce my damage" moments which doesn't feel great, as well as some fight admin.  

I think I'd prefer an increase of how much damage they've already taken, or resistance points they've already drained

Reply:
Yeah I like this idea, could be annoying to calculate the resistance points a creature has lost but yeah I do prefer that